presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,65 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix incorrect query result for queries involving timestamp, when the time zone database
+  embedded in the version of Joda-Time that Presto depends on does not match the time zone
+  database used by the JVM that Presto runs on.
+* Fix count(*) aggregation on empty relation when ``optimize_mixed_distinct_aggregation`` is enabled.
+* Improve performance of table scan on structured types by removing unnecessary memory accounting.
+* Add `reduce_agg` aggregate function.
+* Add :func:`millisecond` function (:issue:`#9524`).
+* Add an optimizer rule to push filters on window function partition symbols under the window
+  operator.
+* Remove ``SHOW STATS ON`` syntax.
+* Restrict ``WHERE`` clause in ``SHOW STATS`` to filters that can be pushed down to the connector.
+* Remove ``node_id`` column from ``system.runtime.queries`` table.
+* Return final results to clients immediately for failed queries.
+
+Web UI
+------
+
+* Fix the rendering of live plan view for queries involving index joins (:issue:`12054`).
+
+Hive Changes
```

Comment:
Our new style is to say "X Connector Changes" for all connectors, so this should be "Hive Connector Changes". Same for the others.